flang/lib/Parser/openmp-parsers.cpp

```diff
@@ -473,6 +473,22 @@ TYPE_CONTEXT_PARSER("Omp LINEAR clause"_en_US,
 // OpenMPv5.2 12.5.2 detach-clause -> DETACH (event-handle)
 TYPE_PARSER(construct<OmpDetachClause>(Parser<OmpObject>{}))
 
+// InteropTypes
+TYPE_PARSER(construct<InteropType>(
+              "TARGETSYNC" >> pure(InteropType::Kind::TargetSync) ||
+              "TARGET" >> pure(InteropType::Kind::Target)))
+
```

Comment:
Remove extra line

Reply:
Addressed the comment on [#1c57d1e](https://github.com/swatheesh-mcw/llvm-project/pull/1/commits/1c57d1ea2b8e7095a834689ad5b832e6a2a84c37)